tests/sentry/spans/consumers/process_segments/test_convert.py

```diff
@@ -36,11 +38,7 @@
         },
         "my.u64.field": 9447000002305251000,
         "my.array.field": [1, 2, ["nested", "array"]],
-    },
-    "measurements": {
-        "num_of_spans": {"value": 50.0},
-        "client_sample_rate": {"value": 0.1},
-        "server_sample_rate": {"value": 0.2},
+        "num_of_spans": 50.0,
```

Comment:
This corresponds to relay's output, see https://github.com/getsentry/relay/pull/5093.